container-host-files/etc/hcf/config/role-manifest.yml

```diff
@@ -1765,6 +1829,7 @@ configuration:
       value_type: certificate
     description: PEM-encoded CA certificate used to sign TLS certificate hosted by Diego BBS.
     required: true
+  - name: JWT_SIGNING_PUB
```

Comment:
Ah; that wasn't clear (because it wasn't `internal`… or was that `type: environment`? I can never keep those two straight in my head)